### PR TITLE
[Android] Converting dimensions to pixels in popup renderer

### PR DESCRIPTION
**Pull request by geskill**
### Bugs Fixed ###

- Fixes #

### API Changes ###

None.

### Behavioral Changes ###

None.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [x] Changes adhere to coding standard

- [ ] Updated [documentation](https://github.com/MicrosoftDocs/xamarin-communitytoolkit)

### CODE REVIEW

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/Android/PopupRenderer.android.cs
Comment (Cfun1) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/Android/PopupRenderer.android.cs:188]: Not sure if `realContentHeight ` will work fine as it was calculated based on `Element.Size.Width`, while here it was `Element.Content.HeightRequest`

**Review by Cfun1 — APPROVED** [review]

**Comment by Cfun1** [comment]
It looks good, tried it on the sample.

**Comment by pictos** [comment]
@ahoefling can you take a look at this one, when it's ready for review?

**Comment by Cfun1** [comment]
@pictos can you rerun the checks from here, I think it an issue in Azure not the code

**Comment by pictos** [comment]
@Cfun1 I don't believe that I have permission to trigger the build. The easiest way to do it is the author of the PR send a dumb commit

**Comment by ahoefling** [comment]
@pictos can you assign me as a reviewer I'll take a look tomorrow morning. Checked from my phone and this appears to make sense

**Comment by pictos** [comment]
@ahoefling for some reason I can't select your name :( 
@jfversluis can you assign Andrew as a reviewer of this PR?

**Comment by pictos** [comment]
@ahoefling I can't assign you as a reviewer, but I can assign this PR to you, not sure if this makes GitHub notify you whatever this PR changes.

**Review by ahoefling — CHANGES_REQUESTED** [review]
The implementation looks great to me! Thank you so much for this because the Popup Size will be similar to iOS and UWP making any sizing math much easier.

When I first ran this in the sample app the popups were being drawn off screen because they were not updated. Please go back and update the sample code. Check all the popup.xaml files for a size and update accordingly. To make this easier there is a `PopupSize.cs` class with constants you can update. That isn't used in all popups, so you'll need to audit the rest.

Here is what it looks like unaltered
image

When I updated the size to `300x300` it looked like it did before. I would push the changes but I am not an official repo maintainer, so I'll need you or another maintainer to get in there and make this change

**Comment by ahoefling** [comment]
This PR is missing the `Popup` label, can we have a maintainer add it

**Comment by geskill** [comment]
I updated the `PopupSize.cs` definitions, like somehow mentioned here https://github.com/xamarin/XamarinCommunityToolkit/pull/839#issuecomment-774756295 .. now we have relative to screen size and fixed size examples.

**Comment by jfversluis** [comment]
Let me know when this is ready to go according to you @ahoefling !

File: samples/XCT.Sample/Pages/Views/Popups/PopupSize.cs
Comment (ahoefling) [inline samples/XCT.Sample/Pages/Views/Popups/PopupSize.cs:14]: I like this, great idea!

**Review by ahoefling — COMMENTED** [review]
This looks good to me and I think we are safe to merge. I am going to download the updates later tonight to do a final check before I approve the PR

**Review by ahoefling — APPROVED** [review]
Just downloaded the code and everything looks good.

**Review by pictos — APPROVED** [review]

**Pull request merged**
